tests/libkvconfig/kvconfig.go

```diff
@@ -0,0 +1,58 @@
+/*
+ * This file is part of the kubevirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package libkvconfig
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
+	v1 "kubevirt.io/api/core/v1"
+
+	"kubevirt.io/kubevirt/pkg/apimachinery/patch"
+
+	"kubevirt.io/kubevirt/tests/framework/kubevirt"
+	"kubevirt.io/kubevirt/tests/util"
+)
+
+func RegisterKubevirtConfigChange(change func(c v1.KubeVirtConfiguration) ([]patch.PatchOperation, error)) {
+	BeforeEach(func() {
```

Comment:
Because we do not need the boilerplate of specifying it. It does the needed to implement the registration.

This has been already raised in another thread.

I see there is opposition to changes like these, although I did ref k8s using the same technique.
To avoid delaying this change, I'll take the ginkgo stuff out.

Done